### PR TITLE
Update readme to have consistency for public repos

### PR DESCRIPTION
**Pull request by bobbytables**
We're working to add some consistency to our public repositories on GitHub. This introduces some minor changes to the top part that centers it and adds our logo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,10 +41,6 @@
 
 ---
 
-**need info?** Slack us at [#team-design-system][slack] ↗️ ⎮ [Notion][notion] ↗️ team page ⎮ [Clubhouse][clubhouse] ↗️ board
```
Comment (bobbytables) [inline README.md:22]: These were removed because the links don't seem to go anywhere and our internal anyways?

Comment (caseymhunt) [reply README.md:22]: They do actually- it's a way to cleanly use repeated links in Markdown. Check out line `142` on for the link source in this file.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,35 @@
...
+    <a href="https://firehydrant.io">
+        <img src="https://img.shields.io/github/v/release/firehydrant/design-system" alt="latest release">
+    </a>
+    <a href="https://firehydrant.io">
+        <img src="https://github.com/firehydrant/design-system/actions/workflows/chromatic.yml/badge.svg" alt="chromatic passing">
+    </a>
+    <a href="https://firehydrant.io">
+        <img src="https://github.com/firehydrant/design-system/actions/workflows/playroom.yml/badge.svg" alt="playroom workflow">
+    </a>
+</p>
```
Comment (caseymhunt) [inline README.md:33]: 😭 I hate mixing HTML with Markdown. So ugly.  That's the only way to get centered content I guess. ¯\_(ツ)_/¯

**Review by caseymhunt — APPROVED** [review]
I leave it to your judgment but if we don't want to include internal links, the Markdown link references (lines `145-147`) to internal sources might need to also be removed.

**Pull request merged**